scripts/setup-services.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+set -e
+
+PG_LOCATION="/usr/lib/postgresql/9.4/bin"
+DB_USERNAME="thimble"
+DB_PASSWORD="thimble"
+ID_WEBMAKER_ENV="POSTGRE_CONNECTION_STRING=postgres://$DB_USERNAME:$DB_PASSWORD@localhost:5432/webmaker_oauth_test HOST=0.0.0.0"
```

Comment:
Which if the apps actually needs these?  If I put them in `ecosystem.json` I have to declare it with the app definition.  Otherwise I can put it in the `start-services.sh` file, which might make sense too.